test/internal/modules/charge-information/routes/charge-element.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+const {
+  experiment,
+  test,
+  beforeEach
+} = exports.lab = require('@hapi/lab').script();
+
+const uuid = require('uuid/v4');
+const { expect } = require('@hapi/code');
+
+const preHandlers = require('internal/modules/charge-information/pre-handlers');
+const routes = require('../../../../../src/internal/modules/charge-information/routes/charge-element');
+const testHelpers = require('../../../test-helpers');
+
+const validChargeElementSteps = [
+  'purpose',
+  'description',
+  'abstraction',
+  'quantities',
+  'time',
+  'source',
+  'season',
+  'loss'
+];
```

Comment:
As mentioned in another test, if you take these as the keys from the `routingConfig` object in the controller, this makes future changes easier